app/Actions/Import/Exec.php

```diff
@@ -165,128 +165,134 @@ private function memWarningCheck()
 	 * @throws ImagickException
 	 */
 	public function do(
-		string $path,
+		string $path_list,
```

Comment:
```suggestion
		array $paths,
```

or

```suggestion
		string $path,
```

I would like to see an proper array of paths here. "Exploding" a list of paths into an array (as below) should - if at all - happen outside of the method. Alternatively, the method might stay as is and be called iteratively for each path by the caller (this is by `FromServer`).

On top, I would actually go one step further and say it is the responsibility of the front-end to already pass a proper JSON array of paths to be imported and not a string separated by semicolons. I.e. the `ImportServerRequest` should not expect a string for `path` but an array for `paths`. This will help to implement other clients in the long run. Please stop the "stringification" of arguments.

Reply:
Fixed.